modules/persistent-storage-csi-migration-vsphere.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-migration-sc-vsphere_{context}"]
+= vSphere automatic migration
+
+=== New installations of {product-title}
+For new installations of {product-title} 4.13, or later, automatic migration is enabled by default.
+
+=== Upgrading {product-title}
+When upgrading from {product-title} 4.12, or earlier, to 4.13, automatic CSI migration for vSphere only occurs if you opt in.
+
+Note the following consequences when opting in to migration:
+
+* Enabling migration cannot be undone.
+
+* Migration can take awhile to complete depending on how many nodes are on the cluster.
+
+* Migration is a disruptive process. The Machine Config Operator (MCO) has to update kubelet, which means degrading each node as it rolls out the new MachineConfig to each machine.
+
+* In some cases, persistent volumes (PVs) created using the vSphere in-tree driver do not attach to nodes after migration.  
```

Comment:
This needs to be called out as a warning (vs a note). This is here as to " recommend customers to carefully review the known issues before opting-in." (however this only lists 1 of the 3 known issues).

Reply:
@sferich888 We cannot use a warning here as per the IBM Style Guide. The closest thing would be an "Attention" notice which requires: "Use an Attention notice to indicate the possibility of damage to a program, device, system, or data." Does this qualify? I've reworded this to make the message more attention getting. PTAL.

Reply:
I think the attention notice is fine; however this doesn't qualify because 'damage' isn't done, what we are really calling attention to is a loss in operational functionality of a customer application (IE: downtime). 

Not sure how far we are allowed to stretch that guidance; but to me an attention call out here is warranted.

Reply:
Done.